packages/feedback/src/core/integration.ts

```diff
@@ -37,238 +37,256 @@ type Unsubscribe = () => void;
 /**
  * Allow users to capture user feedback and send it to Sentry.
  */
-export const feedbackIntegration = (({
-  // FeedbackGeneralConfiguration
-  id = 'sentry-feedback',
-  showBranding = true,
-  autoInject = true,
-  showEmail = true,
-  showName = true,
-  showScreenshot = false,
-  useSentryUser = {
-    email: 'email',
-    name: 'username',
-  },
-  isNameRequired = false,
-  isEmailRequired = false,
 
-  // FeedbackThemeConfiguration
-  colorScheme = 'system',
-  themeLight,
-  themeDark,
-
-  // FeedbackTextConfiguration
-  buttonLabel = ACTOR_LABEL,
-  cancelButtonLabel = CANCEL_BUTTON_LABEL,
-  submitButtonLabel = SUBMIT_BUTTON_LABEL,
-  formTitle = FORM_TITLE,
-  emailLabel = EMAIL_LABEL,
-  emailPlaceholder = EMAIL_PLACEHOLDER,
-  messageLabel = MESSAGE_LABEL,
-  messagePlaceholder = MESSAGE_PLACEHOLDER,
-  nameLabel = NAME_LABEL,
-  namePlaceholder = NAME_PLACEHOLDER,
-  successMessageText = SUCCESS_MESSAGE_TEXT,
-  isRequiredText = IS_REQUIRED_TEXT,
-
-  // FeedbackCallbacks
-  onFormOpen,
-  onFormClose,
-  onSubmitSuccess,
-  onSubmitError,
-  onFormSubmitted,
-}: OptionalFeedbackConfiguration = {}) => {
-  const _options = {
-    id,
-    autoInject,
-    showBranding,
-    isEmailRequired,
-    isNameRequired,
-    showEmail,
-    showName,
-    showScreenshot,
-    useSentryUser,
-
-    colorScheme,
-    themeDark: {
-      ...DEFAULT_THEME.dark,
-      ...themeDark,
-    },
-    themeLight: {
-      ...DEFAULT_THEME.light,
-      ...themeLight,
+interface BuilderOptions {
+  // The type here should be `keyof typeof LazyLoadableIntegrations`, but that'll cause a cicrular
+  // dependency with @sentry/core
+  lazyLoadIntegration: (name: 'feedbackModalIntegration' | 'feedbackScreenshotIntegration') => Promise<IntegrationFn>;
+  getModalIntegration?: null | (() => IntegrationFn);
+  getScreenshotIntegration?: null | (() => IntegrationFn);
+}
+export const buildFeedbackIntegration = ({
+  lazyLoadIntegration,
+  getModalIntegration,
+  getScreenshotIntegration,
+}: BuilderOptions): IntegrationFn => {
+  const feedbackIntegration = (({
+    // FeedbackGeneralConfiguration
+    id = 'sentry-feedback',
+    showBranding = true,
+    autoInject = true,
+    showEmail = true,
+    showName = true,
+    showScreenshot = true,
```

Comment:
Is this intentional to switch to true?

Reply:
This is a new field for v8, so i guess we need to pick the default.
I'll flip it back so that can be an explicit discussion/change.